SmartDeviceLink/SDLOnHMIStatus.h

```diff
@@ -33,6 +34,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (strong, nonatomic) SDLAudioStreamingState audioStreamingState;
 
+/**
+ Current availablility of video streaming for the application. When this parameter is NOT_STREAMABLE, the application must stop video streaming to SDL.
+ */
+@property (strong, nonatomic) SDLVideoStreamingState videoStreamingState;
```

Comment:
Documentation should include that `videoStreamingState` is a required parameter. Should probably add the *mandatory* status to the rest of the params.

Reply:
This will have to be addressed at the SDLC meeting, as this being required is a breaking change.